9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
I'm not worried about `postgres` being upgraded, I'm worried about all the other unrelated upgrades that will happen when running `apk upgrade`.

Here's a diff of running `apk info -v` between the base `postgres:9.6-alpine` image and `mdillon/postgis:9.6-alpine` as built from your branch:

```diff
--- /dev/fd/63	2017-02-06 11:15:50.000000000 -0800
+++ /dev/fd/62	2017-02-06 11:15:50.000000000 -0800
@@ -1,29 +1,42 @@
+.postgis-rundeps-0
 .postgresql-rundeps-0
+WARNING: Ignoring APKINDEX.066df28d.tar.gz: No such file or directory
+WARNING: Ignoring APKINDEX.30e6f5af.tar.gz: No such file or directory
 WARNING: Ignoring APKINDEX.c51f8f92.tar.gz: No such file or directory
 WARNING: Ignoring APKINDEX.d09172fd.tar.gz: No such file or directory
 alpine-baselayout-3.0.4-r0
 alpine-keys-1.3-r0
-apk-tools-2.6.8-r1
-bash-4.3.46-r5
-busybox-1.25.1-r0
-libc-utils-0.7-r1
+apk-tools-2.6.8-r2
+bash-4.3.48-r1
+busybox-1.26.1-r3
+gdal-2.1.0-r1
+geos-3.5.0-r0
+giflib-5.1.4-r1
+json-c-0.12.1-r0
+libc-utils-0.7.1-r0
 libcrypto1.0-1.0.2k-r0
 libedit-20150325.3.1-r3
-libgcrypt-1.7.3-r0
+libgcc-6.3.0-r1
+libgcrypt-1.7.5-r0
 libgpg-error-1.24-r0
-libressl2.4-libcrypto-2.4.4-r0
-libressl2.4-libssl-2.4.4-r0
+libjpeg-turbo-1.5.1-r0
+libpng-1.6.27-r0
+libressl2.4-libcrypto-2.4.5-r0
+libressl2.4-libssl-2.4.5-r0
 libssl1.0-1.0.2k-r0
+libstdc++-6.3.0-r1
 libuuid-2.28.2-r1
 libxml2-2.9.4-r0
 libxslt-1.1.29-r0
-musl-1.1.15-r6
-musl-utils-1.1.15-r6
+musl-1.1.16-r3
+musl-utils-1.1.16-r3
 ncurses-libs-6.0-r7
 ncurses-terminfo-6.0-r7
 ncurses-terminfo-base-6.0-r7
-readline-6.3.008-r4
+proj4-4.9.1-r0
+readline-6.3.008-r5
 scanelf-1.1.6-r0
 su-exec-0.2-r0
-tzdata-2016i-r0
-zlib-1.2.8-r2
+tiff-4.0.7-r1
+tzdata-2016j-r0
+zlib-1.2.11-r0
```

As you can see, there are a bunch of updates that happen that are unrelated to installing the dependencies needed for `postgis`. I'm not comfortable with the idea that upgrading any arbitrary package from the numbered version of Alpine to `edge` will always be safe to do.